### PR TITLE
DOC: docstring examples geom_type, bounds, total_bounds

### PR DESCRIPTION
**Pull request by abonte**
Add example for:

- `geom_type`
- `bounds`
- `total_bounds`

### CODE REVIEW

File: geopandas/base.py [hunk 1]
```diff
@@ -149,8 +149,23 @@ def crs(self, value):
...
-        """Returns a ``Series`` of strings specifying the `Geometry Type` of each
-        object."""
+        """
+        Returns a ``Series`` of strings specifying the `Geometry Type` of each
+        object.
+
+        Examples
+        --------
+        >>> from shapely.geometry import Point, Polygon, LineString
+        >>> d = {'geometry': [Point(), Point(2,1),  Polygon([(0, 0), (1, 1), (1, 0)]),
```
Comment (martinfleis) [inline geopandas/base.py:159]: Can you leave out `Point()` for this example? It is unnecessarily confusing that it changes to empty GeometryCollection. That is not the point of this example.

File: geopandas/base.py [hunk 2]
```diff
@@ -584,6 +611,15 @@ def total_bounds(self):
...
         the series.
+
+        Examples
+        --------
+        >>> from shapely.geometry import Point, Polygon, LineString
+        >>> d = {'geometry': [Point(3,-1), Polygon([(0, 0), (1, 1), (1, 0)]),
+.       ... LineString([(0, 1), (1, 2)])]}
+        >>> gdf = gpd.GeoDataFrame(d, crs="EPSG:4326")
+        >>> gdf.total_bounds
+        aarray([ 0., -1.,  3.,  2.])
```
Comment (martinfleis) [inline geopandas/base.py:622]:
```suggestion
        array([ 0., -1.,  3.,  2.])
```

File: geopandas/base.py (hunk 1 above)
Comment (martinfleis) [inline geopandas/base.py:159]:
```suggestion
        >>> d = {'geometry': [Point(), Point(2, 1),  Polygon([(0, 0), (1, 1), (1, 0)]),
```

File: geopandas/base.py [hunk 3]
```diff
@@ -571,6 +586,18 @@ def bounds(self):
         ``maxy`` values containing the bounds for each geometry.
 
         See ``GeoSeries.total_bounds`` for the limits of the entire series.
+
+        Examples
+        --------
+        >>> from shapely.geometry import Point, Polygon, LineString
+        >>> d = {'geometry': [Point(2,1), Polygon([(0, 0), (1, 1), (1, 0)]),
```
Comment (martinfleis) [inline geopandas/base.py:593]:
```suggestion
        >>> d = {'geometry': [Point(2, 1), Polygon([(0, 0), (1, 1), (1, 0)]),
```

File: geopandas/base.py [hunk 4]
```diff
@@ -584,6 +611,15 @@ def total_bounds(self):
 
         See ``GeoSeries.bounds`` for the bounds of the geometries contained in
         the series.
+
+        Examples
+        --------
+        >>> from shapely.geometry import Point, Polygon, LineString
+        >>> d = {'geometry': [Point(3,-1), Polygon([(0, 0), (1, 1), (1, 0)]),
```
Comment (martinfleis) [inline geopandas/base.py:618]:
```suggestion
        >>> d = {'geometry': [Point(3, -1), Polygon([(0, 0), (1, 1), (1, 0)]),
```

File: geopandas/base.py [hunk 5]
```diff
@@ -149,8 +149,23 @@ def crs(self, value):
...
+        """
+        Returns a ``Series`` of strings specifying the `Geometry Type` of each
+        object.
+
+        Examples
+        --------
+        >>> from shapely.geometry import Point, Polygon, LineString
+        >>> d = {'geometry': [Point(), Point(2,1),  Polygon([(0, 0), (1, 1), (1, 0)]),
+        ... LineString([(0, 0), (1, 1)])]}
+        >>> gdf = gpd.GeoDataFrame(d, crs="EPSG:4326")
```
Comment (jorisvandenbossche) [inline geopandas/base.py:161]:
```suggestion
        >>> gdf = geopandas.GeoDataFrame(d, crs="EPSG:4326")
```

File: geopandas/base.py [hunk 6]
```diff
@@ -571,6 +586,18 @@ def bounds(self):
         ``maxy`` values containing the bounds for each geometry.
 
         See ``GeoSeries.total_bounds`` for the limits of the entire series.
+
+        Examples
+        --------
+        >>> from shapely.geometry import Point, Polygon, LineString
+        >>> d = {'geometry': [Point(2,1), Polygon([(0, 0), (1, 1), (1, 0)]),
+        ... LineString([(0, 1), (1, 2)])]}
+        >>> gdf = gpd.GeoDataFrame(d, crs="EPSG:4326")
```
Comment (jorisvandenbossche) [inline geopandas/base.py:595]:
```suggestion
        >>> gdf = geopandas.GeoDataFrame(d, crs="EPSG:4326")
```

File: geopandas/base.py [hunk 7]
```diff
@@ -584,6 +611,15 @@ def total_bounds(self):
 
         See ``GeoSeries.bounds`` for the bounds of the geometries contained in
         the series.
+
+        Examples
+        --------
+        >>> from shapely.geometry import Point, Polygon, LineString
+        >>> d = {'geometry': [Point(3,-1), Polygon([(0, 0), (1, 1), (1, 0)]),
+.       ... LineString([(0, 1), (1, 2)])]}
+        >>> gdf = gpd.GeoDataFrame(d, crs="EPSG:4326")
```
Comment (jorisvandenbossche) [inline geopandas/base.py:620]:
```suggestion
        >>> gdf = geopandas.GeoDataFrame(d, crs="EPSG:4326")
```

**Comment by martinfleis** [comment]
Thank you @abonte!

**Pull request merged**